app/sources/colombia.py

```diff
@@ -19,6 +19,8 @@
 
 
 def _get_remote_award(proceso_de_compra: str, proveedor_adjudicado: str) -> tuple[list[dict[str, str]], str]:
+    # To avoid '&' been taken as a query parameter
+    proveedor_adjudicado = proveedor_adjudicado.replace("&", "%26")
```

Comment:
Shouldn't we escape all using urllib?

Reply:
I tried that but for some reason, it wasn't working, the API returns an empty response.

Reply:
Full context: the final URL with urllib is, for example: https://www.datos.gov.co/resource/p6dx-8zbt.json?$where=id_del_portafolio='CO1.BDOS.5572587' AND nombre_del_proveedor='David%20Alexander%20Rodr%C3%ADguez%20Cosme' if you try that URL in your browser it works, however, from the python code the API returns a empty array []. I'm not sure if we are encoding the URL again at some point and that is why it fails from the code.

Reply:
Indeed, that did the trick!

Reply:
Or not, according to the tests..

Reply:
Ah, the single quotes need to be %27. Maybe `parse.quote` the entire query string?

Reply:
It works when I run the command manually so it is weird that is failing in the tests, but I will try that

Reply:
I think we need to mock up the _get_award method from that test anyway. So I will do both